grpc.go

```diff
@@ -0,0 +1,167 @@
+package openmock
+
+import (
+	"encoding/binary"
+	"fmt"
+	"io/ioutil"
+	"time"
+
+	"github.com/labstack/echo/v4"
+	"github.com/labstack/echo/v4/middleware"
+	"github.com/sirupsen/logrus"
+	"golang.org/x/net/http2"
+	"google.golang.org/protobuf/encoding/protojson"
+	"google.golang.org/protobuf/proto"
+)
+
+const (
+	grpcPayloadLen = 1
+	grpcSizeLen    = 4
+	grpcHeaderLen  = grpcPayloadLen + grpcSizeLen
+)
+
+// length-prefixed message, see https://github.com/grpc/grpc/blob/master/doc/PROTOCOL-HTTP2.md
+func msgHeader(data []byte) (hdr []byte, payload []byte) {
+	hdr = make([]byte, grpcHeaderLen)
+
+	hdr[0] = byte(0)
+
+	// Write length of payload into buf
+	binary.BigEndian.PutUint32(hdr[grpcPayloadLen:], uint32(len(data)))
+	return hdr, data
+}
+
+// GRPCService is a map of service_name => GRPCRequestResponsePair
+type GRPCService map[string]GRPCRequestResponsePair
+
+// GRPCRequestResponsePair is a pair of proto.Message to define
+// the message schema of request and response of a method
+type GRPCRequestResponsePair struct {
+	Request  proto.Message
+	Response proto.Message
+}
+
+func (om *OpenMock) convertJSONToH2Response(ctx Context, resJSON string) (header []byte, data []byte, err error) {
```

Comment:
moved the convert JSON <=> protobuf struct functions here.

Reply:
makes sense, this is cleaner 👍